src/d3d12/D3D12.h

```diff
@@ -3,6 +3,8 @@
 #include "common/D3D12Downlevel.h"
 #include "window/Window.h"
 
+extern IMGUI_IMPL_API LRESULT ImGui_ImplWin32_WndProcHandler(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam);
```

Comment:
this should not be required if you update it right, unsure why you need it

Reply:
This is because of how the new Win32 impl is written. It's wrapped in `#IF 0`, and I just put it here since it's where it's used. See: https://github.com/maximegmd/CyberEngineTweaks/pull/970/files/dea67f4af117072d46e8b799c4a1770f9c550726#diff-e3de0213e18b0bd55bf53f2687e616d36dd853e460ba511e634cae6c4671e981L28

Reply:
as pointed out by another comment, just change it to 1 or remove please